doc/tutorials/introduction/linux_install/linux_install.markdown

```diff
@@ -98,7 +98,7 @@ Building OpenCV from Source Using CMake
         will be statically linked with OpenCV and can take a lot of memory.
 
 -#  [optional] Generate pkg-config info
-    -   Add this flag when running CMake: `-DOPENCV_GENERATE_PKGCONFIG=ON`
+    -   Add this flag when running CMake: `-D OPENCV_GENERATE_PKGCONFIG=ON`
```

Comment:
Interesting. I thought the space between the D and option token was optional for CMake when specified on the CLI. When I use -DOPENCV_GENERATE_PKGCONFIG=On as part of the initial cmake configuration command, and open up the CMakeCache in the cmake-gui, it shows the option as set. Subsequently running cmake -DOPENCV_GENERATE_PKGCONFIG=Off disables pkgconfig generation in cmake-gui .